src/main/java/org/aksw/simba/squirrel/analyzer/mime/TypeDetector.java

```diff
@@ -9,7 +9,7 @@
  * This interface defines the functionality to detect the mime-types, especially
  * of RDF serializations
  *
- * @author Abhishek Hassan Chandrashekar (abhihc@mail.uni-paderborn.de)
+ * @author Abhishek Hassan Chandrashekar&ouml;der (abhihc@mail.uni-paderborn.de)
```

Comment:
looks like something went wrong here :smile:

Reply:
Corrected the typo mistake